### PR TITLE
Fix integration tests issues

### PR DESCRIPTION
**Pull request by palaciosjeremias**
|Related issue|
|---|
|5955|

## Description
This PR include fixes for the following issues found during integration tests development:
-wdb_global_update_agent_name
-sync-agent-info-get query
-Unitest fix for wdb_global_update_agent_name changes
-Renaming of wdb_global_sync_agent_info_get
-Renaming of start_id parameters

## Tests

- Compilation without warnings in every supported platform
  - [X] Linux
- [X] Source installation

- Memory tests for Linux
  - [X] Valgrind (memcheck and descriptor leaks check)

### CODE REVIEW

File: src/wazuh_db/wdb_global.c [hunk 1]
```diff
@@ -105,12 +105,12 @@ int wdb_global_update_agent_name(wdb_t *wdb, int id, char* name) {
 
     stmt = wdb->stmt[WDB_STMT_GLOBAL_UPDATE_AGENT_NAME];
 
-    if (sqlite3_bind_int(stmt, 1, id) != SQLITE_OK) {
-        merror("DB(%s) sqlite3_bind_int(): %s", wdb->id, sqlite3_errmsg(wdb->db));
+    if (sqlite3_bind_text(stmt, 1, name, -1, NULL) != SQLITE_OK) {
```
Comment (pereyra-m) [inline src/wazuh_db/wdb_global.c:108]: Good fix!

**Review by pereyra-m — APPROVED** [review]
LGTM!

File: src/wazuh_db/wdb.c [hunk 2]
```diff
@@ -143,7 +143,7 @@ static const char *SQL_STMT[] = {
     [WDB_STMT_GLOBAL_DELETE_GROUP] = "DELETE FROM `group` WHERE name = ?;",
     [WDB_STMT_GLOBAL_SELECT_GROUPS] = "SELECT name FROM `group`;",
     [WDB_STMT_GLOBAL_SELECT_AGENT_KEEPALIVE] = "SELECT last_keepalive FROM agent WHERE name = ? AND (register_ip = ? OR register_ip LIKE ? || '/_%');",
-    [WDB_STMT_GLOBAL_SYNC_REQ_GET] = "SELECT id, name, ip, os_name, os_version, os_major, os_minor, os_codename, os_build, os_platform, os_uname, os_arch, version config_sum, merged_sum, manager_host, node_name, last_keepalive FROM agent WHERE id > ? AND sync_status = 1 LIMIT 1;", 
+    [WDB_STMT_GLOBAL_SYNC_REQ_GET] = "SELECT id, name, ip, os_name, os_version, os_major, os_minor, os_codename, os_build, os_platform, os_uname, os_arch, version, config_sum, merged_sum, manager_host, node_name, last_keepalive FROM agent WHERE id > ? AND sync_status = 1 LIMIT 1;",
```
Comment (DProvinciani) [inline src/wazuh_db/wdb.c:146]: 🤦

File: src/wazuh_db/wdb_global.c (hunk 1 above)
Comment (DProvinciani) [reply src/wazuh_db/wdb_global.c:108]: +1

**Review by DProvinciani — APPROVED** [review]
Looks good!!

**Pull request merged**